serving/docs/lmi/user_guides/release_notes.md

```diff
@@ -0,0 +1,64 @@
+# LMI V10 containers release
+
+This document will contain the latest releases of our LMI containers. For details on any other previous releases, please refer our [github release page](https://github.com/deepjavalibrary/djl-serving/releases)
+
+## Release Notes
+
+### Release date: June 6, 2024
+
+Check out our latest [Large Model Inference Containers](https://github.com/aws/deep-learning-containers/blob/master/available_images.md#large-model-inference-containers).
+
+### Key Features
+
+#### LMI container
+
+- Provided general performance optimization.
+- **Added text embedding support**
+  - Our solution for text embedding is 5% faster than HF TEI solution.
+- Multi-LoRA feature now supports LLama3 and AWS models
+
+#### TensorRT-LLM container
+
+- Upgraded to TensorRT-LLM 0.9.0
+- AWQ, FP8 support for Llama3 models on G6/P5 machines
+- Now, default max_new_tokens=16384
+- Bugfix for critical memory leaks on long run. 
+- Bugfix for model hanging issues.
+
+#### Transformers NeuronX container
+
+- Upgraded to Transformers NeuronX 2.18.2
+
+#### DeepSpeed container (deprecated)
+
+- DeepSpeed container is now deprecated. If you are not using `deepspeed` engine, all you need is `0.28.0-lmi` container and continue using it. 
```

Comment:
Can we say "We have removed support for deepspeed and renamed our deepspeed container to lmi. The lmi container contains lmi-dist and vllm, and all existing workloads with deepspeed can be easily migrated to one of these backends. See https://github.com/deepjavalibrary/djl-serving/blob/master/serving/docs/lmi/announcements/deepspeed-deprecation.md for steps on how to migrate your workload".

Reply:
Done